web/apps/labelstudio/src/app/App.jsx

```diff
@@ -17,18 +18,30 @@ import "./App.styl";
 import { AsyncPage } from "./AsyncPage/AsyncPage";
 import ErrorBoundary from "./ErrorBoundary";
 import { RootPage } from "./RootPage";
-import { FF_OPTIC_2, isFF } from "../utils/feature-flags";
+import { FF_OPTIC_2, FF_UNSAVED_CHANGES, isFF } from "../utils/feature-flags";
 import { ToastProvider, ToastViewport } from "../components/Toast/Toast";
 
 const baseURL = new URL(APP_SETTINGS.hostname || location.origin);
+export const UNBLOCK_HISTORY_MESSAGE = "UNBLOCK_HISTORY";
 
 const browserHistory = createBrowserHistory({
   basename: baseURL.pathname || "/",
   getUserConfirmation: (message, callback) => {
```

Comment:
what is the purpose of `callback`? is it used to unblock navigation?

Reply:
Yep. And by calling it with false/true we decide should we prevent going to another "page" or not.

Reply:
so can you rename it to reflect semantics? or at least add a comment please